Add ZWA-2 as recommended Z-Wave adapter (#40494)
home-assistant/home-assistant.github.io: source/_docs/z-wave/controllers.markdown

    title: Home Assistant Connect ZWA-2
---

> To run Z-Wave on Home Assistant, you need to have a compatible Z-Wave adapter installed.

Author: ```suggestion
To use Z-Wave with Home Assistant, you need a compatible Z-Wave adapter.
```

## Recommended Z-Wave adapter

- [Home Assistant Connect ZWA-2](/connect/zwa-2/)

> ## Supported Z-Wave adapters

Reviewer: Maybe? ZWA-2 is also supported and not listed. 
```suggestion
## Other supported Z-Wave adapters
```
